binding_generator.py

```diff
@@ -321,7 +321,7 @@ def escape_default_arg(_type, default_value):
 
 
 
-            if argument["has_default_value"] or has_default_argument:
+            if (argument["has_default_value"] and argument["default_value"] != "") or has_default_argument:
```

Comment:
I'm not sure that's the proper fix, the issue is that the default value is empty because Godot's API generator is buggy (doesn't handle NodePath properly).

Might still be worth raising an error here and handling it gracefully.

Reply:
I'll handle it nicer in a separate PR